app/javascript/app/utils/uswds.js

```diff
@@ -0,0 +1,31 @@
+// This file serves as a minimal, progressively-enhanced entry point for the
+// Login.gov Design System component integration. It serves two purposes:
+//
+// 1. Require explicit opt-in to components, to avoid behaviors from being
+//    applied to legacy (ad hoc) implementations.
+// 2. Import the minimal behavior code for components used in the application.
+//
+// The specific code (notably polyfills and configuration) follows that of the
+// default entry point for the design system, which can serve as reference.
+//
+// See: https://github.com/18F/identity-style-guide/blob/master/src/js/main.js
+// See: https://github.com/uswds/uswds/blob/develop/src/js/start.js
+
+import domready from 'domready';
+import 'uswds/src/js/polyfills';
+import 'uswds/src/js/config';
+import accordion from 'uswds/src/js/components/accordion';
+
+const COMPONENTS = {
+  '.usa-banner': accordion,
+};
+
+domready(() => {
+  Object.keys(COMPONENTS).forEach((selector) => {
+    const behavior = COMPONENTS[selector];
+    const target = document.querySelector(selector);
```

Comment:
`querySelector` returns only a single instance... do we want to look for all `.usa-banner` on a page? Or should we call `behavior.on(document)` instead?

From https://designsystem.digital.gov/documentation/developers/#js-customization:
> `on`: Initialize a component’s JavaScript behavior by passing the root element, such as `window.document`.

Reply:
> Or should we call `behavior.on(document)` instead?

This ties into my comment at https://github.com/18F/identity-idp/pull/3957/files#r459094095 . Yes, ideally we don't have to cherry-pick elements from the page, but instead initialize all components in the entire document. In the end, maybe it'd be less effort and less error-prone to do this from the start.

Reply:
Re: that other comment, I don't have strong opinions on cherry-picking or doing the whole page, both seem acceptable to me.

My main worry here is that if we happen to have two `.usa-banner` on a page, this will ignore the second one, that's the only thing I'd like to see addressed. Otherwise if we want to be clear it's only one, should we throw on an id? `#usa-banner` etc?

Reply:
It's a good point, and while it might not be quite as problematic for the banner specifically, once we apply this to other components, it's much more likely to anticipate there being multiple elements matching the selector. So if we decide to keep this, it should definitely support multiple results.

Reply:
Ok so should we just change to `.querySelectorAll(...).forEach()`?